pkg/controller/postgres/postgres_controller.go

```diff
@@ -24,11 +24,15 @@ import (
 	contrail "github.com/Juniper/contrail-operator/pkg/apis/contrail/v1alpha1"
 	"github.com/Juniper/contrail-operator/pkg/certificates"
 	"github.com/Juniper/contrail-operator/pkg/controller/utils"
-	"github.com/Juniper/contrail-operator/pkg/volumeclaims"
+	"github.com/Juniper/contrail-operator/pkg/k8s"
+	contraillabel "github.com/Juniper/contrail-operator/pkg/label"
+	"github.com/Juniper/contrail-operator/pkg/localvolume"
 )
 
 var log = logf.Log.WithName("controller_postgres")
 
+const defaultPostgresStoragePath = "/mnt/postgres"
```

Comment:
What do you think about having similar path as zookeeper and Cassandra?

You can change it here or change in zookeeper and cassandra

Reply:
we thought about that but I don't want to use /var/lib here I would change that in Cassandra and Zookeeper, can it be done in a next PR to do not increase amount of changes?

Reply:
ok